defaults/main.yml

```diff
@@ -2,9 +2,53 @@
 
 ## General
 elasticsearch_version: 7.5.2
```

Comment:
You could put 7.6.0. It's the current release :nerd_face:

Reply:
Its a good point check the breaking changes when upgrading to a minor release. May be necessary to change things and rerun tests!
https://www.elastic.co/guide/en/elasticsearch/reference/current/rolling-upgrades.html
https://www.elastic.co/guide/en/elasticsearch/reference/current/release-notes-7.6.0.html